sdks/python/gen_protos.py

```diff
@@ -145,7 +145,8 @@ def _install_grpcio_tools_and_generate_proto_files():
         'Installing grpcio-tools took %0.2f seconds.' % (time.time() - start))
   finally:
     sys.stderr.flush()
-    shutil.rmtree(build_path)
+    if os.path.exists(build_path):
+      shutil.rmtree(build_path)
```

Comment:
Should we instead to `shutil.rmtree(build_path, ignore_errors=True)` to all sorts of clean up errors here? Otherwise this look good.